src/js/va-letters/containers/ConfirmAddress.jsx

```diff
@@ -1,19 +1,79 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import { updateAddressConfirmation } from '../actions';
+
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+
 class ConfirmAddress extends React.Component {
+  constructor(props) {
+    super(props);
+    this.confirmAddress = this.confirmAddress.bind(this);
+  }
+
+  confirmAddress() {
+    this.props.onConfirmationOfAddress(true);
+    this.props.router.push('/download-letters');
+  }
+
   render() {
+    const buttons = this.props.letters.addressConfirmed !== false ?
```

Comment:
Re: accessibility, you might try giving each `ProgressButton` an `aria-labelledby` attribute and give it the id of the element holding "Is this address correct?": https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

Or just put in a TODO to that effect for now.